platform/journal.go

```diff
@@ -112,6 +117,14 @@ func (j *Journal) Start(ctx context.Context, m Machine) error {
 	return nil
 }
 
+func (j *Journal) ReadJournal() ([]byte, error) {
```

Comment:
Probably needs a comment that there are no guarantees about the returned bytes if called before `Destroy()`.

Maybe just call this `Read()`?  `journal.ReadJournal` seems redundant.

Reply:
My initial thought for calling it `ReadJournal` was to allow for distinctions if we ever added a `ReadRawJournal` but `Read` & `ReadRaw` would work as well.